packages/redux/src/entities/types/returnOptions.types.ts

```diff
@@ -1,25 +1,9 @@
-import type { MerchantEntity } from './merchant.types';
-import type {
-  MerchantOrderReturnOptions,
-  ReturnOption,
-} from '@farfetch/blackout-client';
+import type { MerchantOrderReturnOptions } from '@farfetch/blackout-client';
 
-export type ReturnOptionEntity = ReturnOption & {
-  merchant: MerchantOrderReturnOptions['merchantId'];
-  merchantOrderId: MerchantOrderReturnOptions['merchantOrderId'];
-  id: string;
-};
+export type ReturnOptionEntity = MerchantOrderReturnOptions;
 
-export type ReturnOptionEntityDenormalized = Omit<
-  ReturnOptionEntity,
-  'merchant'
-> & {
-  merchant: MerchantEntity;
-};
+export type ReturnOptionEntityDenormalized = Array<ReturnOptionEntity>;
```

Comment:
This type does not seem right and it is not being used anywhere. Can you take a look at it?

Reply:
Yep it was wrong, it was now fixed, please validate.